leobot_control/launch/teleop.launch

```diff
@@ -0,0 +1,16 @@
+<?xml version="1.0"?>
+<launch>
+  <arg name="joy_dev" default="/dev/input/js0" />
+  <arg name="joystick" default="true" />
+
+  <group ns="joy_teleop">
+    <rosparam command="load" file="$(find leobot_control)/config/common/teleop.yml" />
```

Comment:
Parameters for joystick teleoperation should be loaded from separate params file. Teleop is too general name. It will be also keyboard teleoperation. Please change it to something related to joystick